anautilities.py

```diff
@@ -66,7 +66,7 @@ def getDirByAnaType(anaType, cName, ztrim=4):
         pass
 
     # Check Paths
-    from ..utils.wrappers import envCheck
+    from gempython.utils.wrappers import envCheck
```

Comment:
Simultaneous changes, but the others were merged in earlier, can you revert yours, or make them conform to the current `develop`? (also L128, L335)

Reply:
If you're talking about commit `2ac2945`, read again: this *reverts* your change (which crashed the program). This PR is based on today's `develop`.

Context: `anautilities.py` is installed in `gempython/gemplotting/utils`, while `wrappers.py` is in `gempython/utils`

Reply:
ah, ok, then the relative import should be `...utils.wrappers`

Reply:
For my own education: `N` leading dots are translated into `N-1` `../` relative to the current file ?

Reply:
For [reference](https://www.python.org/dev/peps/pep-0328/)